portal-kernel/src/com/liferay/portal/kernel/repository/DocumentRepository.java

```diff
@@ -68,21 +68,11 @@ public void checkInFileEntry(
 			String changeLog, ServiceContext serviceContext)
 		throws PortalException;
 
-	public default void checkInFileEntry(
+	public void checkInFileEntry(
 			long userId, long fileEntryId,
 			DLVersionNumberIncrease dlVersionNumberIncrease, String changeLog,
 			ServiceContext serviceContext)
-		throws PortalException {
-
-		boolean majorVersion = false;
-
-		if (dlVersionNumberIncrease == DLVersionNumberIncrease.MAJOR) {
-			majorVersion = true;
-		}
-
-		checkInFileEntry(
-			userId, fileEntryId, majorVersion, changeLog, serviceContext);
-	}
+		throws PortalException;
```

Comment:
Hi @dantewang @ChiZelin,
I can not understand why you remove the usages directly.
According to your next commit, all the methods you want to remove later have replacements, you should replace the usages here rather than removing.

Tina

Reply:
@tinatian No, they have to be removed. The deprecated ones are replaced by the `default` ones, and exactly these `default` ones are delegating to the deprecated ones. We can't replace these usages because the replacements are the `default` methods themselves.

I think they add the `default` methods and let them delegate to deprecated ones to maintain backwards compatibility which allows code implementing just the deprecated methods can transparently work. But since we're removing deprecated methods, introducing breaking changes, we have to force developers implement the replacement methods.